### PR TITLE
Bump version for ecto dependency requirement to 2.2.x

### PR DESCRIPTION
**Pull request by kcurtin**
The `mix.lock` requires ecto `2.1.x`, which prevents upgrading to ecto `2.2.x`. I bumped the version requirement and ran the tests and things seem fine.

### CODE REVIEW

**Comment by matthewlehner** [comment]
It might be better to set the version to `~>2.2` as this will allow for any `2.x` release of Ecto. Since Ecto follows semver, there won't be any breaking changes on minor version changes, just feature additions.

**Comment by leifg** [comment]
I agree, we should switch the version to `~> 2.1`.

As it seems that ecto doesn't support 1.3 any more we could also consider changing the `.travis.yml`. And then bump up the version of `ecto_enum` to `1.1`.

**Comment by kcurtin** [comment]
I updated the dep specification and changed the travis build to run w. elixir 1.4.0 and 1.5.0. I also updated the version to `1.1.0`

cc @gjaldon

**Comment by wfgilman** [comment]
@gjaldon Any chance we can get this PR merged?

**Comment by ryanwinchester** [comment]
Have you guys got this working in the latest ecto/phoenix versions?

I get

    ** (ArgumentError) invalid or unknown type MyEnum for field :my_field

File: mix.exs [hunk 1]
```diff
@@ -31,7 +31,7 @@ defmodule EctoEnum.Mixfile do
   end
 
   defp deps do
-    [{:ecto, "~> 2.1.0"},
+    [{:ecto, "~> 2.2"},
      {:postgrex, "~> 0.13.0", optional: true},
      {:mariaex, "~> 0.8.0", optional: true},
```
Comment (ryanwinchester) [inline mix.exs]: I think

```elixir
{:postgrex, "~> 0.13.0", optional: true},
{:mariaex, "~> 0.8.0", optional: true},
```

should be 

```elixir
```elixir
{:postgrex, "~> 0.13", optional: true},
{:mariaex, "~> 0.8", optional: true},
```

**Pull request merged**